src/components/changePlanner/CPDetailView.tsx

```diff
@@ -164,6 +164,23 @@ class CPDetailView extends React.Component<
       });
   }
 
+  handleExecute() {
+    axios
+      .post(
+        API_ROOT +
+        "/api/change-plans/" +
+        this.state.changePlan.id +
+        "/execute",
+        getHeaders(this.props.token)
+      )
+      .then(res => {
+        this.addSuccessToast(res.data.success_message);
+      })
+      .catch(err => {
+        this.addErrorToast(err.response.data.failure_message);
+      });
+  }
+
```

Comment:
Added this method (all other changes are linting)

Reply:
did you test this? I think for post requests, the default second argument is the body, and the third is the headers. if no body you can just pass in {}

Reply:
THAT EXPLAINS IT